frontend/tests/components/profile/profile-content.test.tsx

```
@@ -0,0 +1,698 @@
+      fireEvent.click(screen.getByText("Friends"));
+      expect(screen.getByText("No friends yet.")).toBeInTheDocument();
+    });
+  });
+
+  describe("Edge Cases", () => {
+    it("handles undefined droplet in enrollment", () => {
+      const enrollmentWithoutDroplet: Enrollment[] = [
+        {
+          id: "2",
+          authorizedUser: mockUserData,
+          droplet: undefined as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)